NEWS.md

```diff
@@ -0,0 +1,303 @@
+- CI, scan builds, release builds etc now use Github actions.
+- Lots of documentation improvements and fixes.
+- Web site updated with new auto complete, better section ids for linking, dark mode, etc. @itchyny #2628
+- Release builds for:
+  - Linux `amd64`, `arm64`, `armel`, `armhf`, `i386`, `mips`, `mips64`, `mips64el`, `mips64r6`, `mips64r6el`, `mipsel`, `mipsr6`, `mipsr6el`, `powerpc`, `ppc64el`, `riscv64` and `s390x`
+  - macOS `amd64` and `arm64`
+  - Windows `i386` and  `amd64`
+  - Docker `linux/386`, `linux/amd64`, `linux/arm64`, `linux/mips64le`, `linux/ppc64le`, `linux/riscv64` and `linux/s390x`
+- Docker images are now available from `ghcr.io/jqlang/jq` instead of docker hub.
+- OSS-fuzz.
+
+Full commit log can be found at https://github.com/jqlang/jq/compare/jq-1.6...jq-1.7 but here are some highlights:
+
+## CLI changes
+
+- Make object key color configurable using `JQ_COLORS` environment variable. @itchyny @haguenau @ericpruitt #2703
+  ```sh
+  # this would make "field" yellow (33, the last value)
+  $ JQ_COLORS="1;30:0;37:0;37:0;37:0;32:1;37:1;37:1;33" ./jq -n '{field: 123}'
+  {
+    "field": 123
+  }
+  ```
+- Respect `NO_COLOR` environment variable to disable color output. See https://no-color.org for details. @itchyny #2728
+- Improved `--help `output. Now mentions all options and nicer order. @itchyny #2747 #2766
+- Last output value can now control exit code using `--exit-code`/`-e`. #1697 @ryo1kato
+  ```sh
+  # true-ish last output value exits with zero
+  $ jq -ne true ; echo $?
+  true
+  0
+  # false-ish last output value (false and null) exits with 1
+  $ jq -ne false ; echo $?
+  false
+  1
+  # no output value exists with 4
+  $ jq -ne empty ; echo $?
+  4
+  ```
+- Add `--binary`/`-b` on Windows for binary output. To get `\n` instead of `\r\n` line endings. 0dab2b1 @nicowilliams
+- Add `--raw-output0` for NUL (zero byte) separated output. @asottile @pabs3 @itchyny #1990 #2235 #2684
+  ```sh
+  # will output a zero byte after each output
+  $ jq -n --raw-output0 '1,2,3' | xxd
+  00000000: 3100 3200 3300                           1.2.3.
+  # can be used with xargs -0
+  $ jq -n --raw-output0 '"a","b","c"' | xargs -0 -n1
+  a
+  b
+  c
+  $ jq -n --raw-output0 '"a b c", "d\ne\nf"' | xargs -0 printf '%q\n'
+  'a b c'
+  'd'$'\n''e'$'\n''f'
+  # can be used with read -d ''
+  $ while IFS= read -r -d '' json; do
+  >   jq '.name' <<< "$json"
+  > done < <(jq -n --raw-output0 '{name:"a b c"},{name:"d\ne\nf"}')
+  "a b c"
+  "d\ne\nf"
+  ```
+  # also it's an error to output a string containing a NUL when using NUL separator
+  $ jq -n --raw-output0 '"\u0000"'
+  jq: error (at <unknown>): Cannot dump a string containing NUL with --raw-output0 option
+- Fix assert crash and validate JSON for `--jsonarg`. @wader #2658
+- Remove deprecated `--argfile` option. @itchyny #2768
+- Rename `--nul-output` to `--raw-output0`, abort on string containing NUL. @itchyny #2684
+
+## Langauge changes
```

Comment:
s/Langauge/Language/

Reply:
Sigh. check-spelling would catch this 😉

Reply:
Noticed now that it actually did but i didn't catch it :)